reactor-core/src/test/java/reactor/core/publisher/FluxBufferUntilChangedTest.java

```diff
@@ -0,0 +1,51 @@
+package reactor.core.publisher;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+import org.junit.Test;
+import reactor.test.StepVerifier;
+
+public class FluxBufferUntilChangedTest {
```

Comment:
can you integrate these tests into `FluxBufferPredicateTest`, with a prefix? Also maybe this needs at least one test with an emphasis on the cleanup of the key 🤔